### PR TITLE
geojson: init at 2.3.0

### PR DESCRIPTION
**Pull request by oxzi**
###### Motivation for this change
Adds the [geojson](https://github.com/frewsxcv/python-geojson) Python-module to nixpkgs

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/geojson/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "geojson";
+  version = "2.3.0";
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/geojson/default.nix]: The `name` attribute is added by `buildPython*` and should therefore be removed.

File: pkgs/development/python-modules/geojson/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "06ihcb8839zzgk5jcv18kc6nqld4hhj3nk4f3drzcr8n8893v1y8";
+  };
+
+  # test_unicode_properties fails for Python 3
```
Comment (FRidh) [inline pkgs/development/python-modules/geojson/default.nix]: Could you post the full error.

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/geojson/default.nix (hunk 2 above)
Comment (geistesk) [reply pkgs/development/python-modules/geojson/default.nix]: Here you are:
```
======================================================================
ERROR: test_unicode_properties (test_features.FeaturesTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/geojson-2.3.0/tests/test_features.py", line 41, in test_unicode_properties
    obj = geojson.load(file_)
  File "/build/geojson-2.3.0/geojson/codec.py", line 43, in load
    **kwargs)
  File "/nix/store/96wn2gz3mwi71gwcrvpfg39bsymd7gqx-python3-3.6.5/lib/python3.6/json/__init__.py", line 296, in load
    return loads(fp.read(),
  File "/nix/store/96wn2gz3mwi71gwcrvpfg39bsymd7gqx-python3-3.6.5/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 28: ordinal not in range(128)

----------------------------------------------------------------------
```

Comment (FRidh) [reply pkgs/development/python-modules/geojson/default.nix]: This is a common issue described in the Nixpkgs manual:

> Unicode issues can typically be fixed by including glibcLocales in buildInputs and exporting LC_ALL=en_US.utf-8.

Comment (geistesk) [reply pkgs/development/python-modules/geojson/default.nix]: I didn't knew about this one. Thanks for your help!

**Comment by FRidh** [comment]
@GrahamcOfBorg build python.pkgs.geojson python3.pkgs.geojson

**Pull request merged**